Configurations/unix-Makefile.tmpl

```diff
@@ -74,6 +76,7 @@ FIPSKEY={- $config{FIPSKEY} -}
 
 VERSION={- "$config{full_version}" -}
 VERSION_NUMBER={- "$config{version}" -}
+RELEASE_DATE={- Time::Piece->strptime($config{"release_date"}, "%d %b %Y")->strftime("%Y-%m-%d") -}
```

Comment:
For devs that build between releases, this will break.  A more complete alternative is this:

```suggestion
RELEASE_DATE={- my $t = $config{"release_date"}
                      ? Time::Piece->strptime($config{"release_date"}, "%d %b %Y")
                      : localtime;
                $t->strftime("%Y-%m-%d") -}
```